ts/features/idpay/initiative/configuration/screens/AssociationSuccessScreen.tsx

```diff
@@ -0,0 +1,66 @@
+import { useNavigation } from "@react-navigation/native";
+import { Text, View } from "native-base";
+import React from "react";
+import { SafeAreaView, StyleSheet } from "react-native";
+import CompletedIcon from "../../../../../../img/pictograms/payment-completed.svg";
+import ButtonDefaultOpacity from "../../../../../components/ButtonDefaultOpacity";
+import { H3 } from "../../../../../components/core/typography/H3";
+import { IOStyles } from "../../../../../components/core/variables/IOStyles";
+import I18n from "../../../../../i18n";
+import themeVariables from "../../../../../theme/variables";
+
+const AssociationSuccessScreen = () => {
+  const navigation = useNavigation();
+
+  // TODO get initative name and ID
+  const initiativeName = "l'Iniziativa 1";
+
+  const handleNavigateToInitiativePress = () => {
+    // TODO handle navigation to initiative details
+    navigation.goBack();
+  };
+
+  return (
+    <SafeAreaView style={IOStyles.flex}>
+      <View style={[IOStyles.horizontalContentPadding, styles.container]}>
+        <View style={styles.content}>
+          <CompletedIcon width={80} height={80} />
```

Comment:
Have you tried to use the new `<Pictogram>` component instead of the direct path to the asset? If not, you can view the **_UI Showroom_** for reference.
```suggestion
          <Pictogram name="completed" size={80} />
```